plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.tsx

```diff
@@ -68,15 +69,14 @@ const TopologyResourcesTabPanel = ({
                 />
               </span>
               <span style={{ flex: '1' }}>
-                {' '}
                 <ResourceStatus
                   additionalClassNames="hidden-xs"
                   noStatusBackground
                 >
-                  <Status status={pod.status?.phase ?? ''} />
+                  <Status status={podPhase(pod)} />
                 </ResourceStatus>
               </span>
-              <span style={{ flex: '1' }}>
```

Comment:
@debsmita1 Why was the style removed?

Reply:
because the `View logs` almost overlapped the pod status. Now I have added some padding instead of removing the flex property